forgettable-frontend/src/pages/edit/EditPerson.js

```diff
@@ -0,0 +1,272 @@
+import {Avatar, Input} from '@mui/material';
+import React from 'react';
+import {useState, useRef} from 'react';
+import CustomButton from '../../components/CustomButton/CustomButton';
+import classes from './EditPerson.module.css';
+import {convertSocialMediaToIcon} from '../../functions/socialMediaIconConverter';
+import {useNavigate, useLocation, useParams} from 'react-router-dom';
+import imageToBase64 from 'image-to-base64/browser';
+import CustomModal from '../../components/CustomModal/CustomModal';
+import * as apiCalls from '../../services/index';
+
+const MAX_IMAGE_SIZE = 16000000;
+
+
+export default function EditPerson() {
+  // to check if it is create person or edit person
+  const location = useLocation();
+
+  const navigate = useNavigate();
+
+  // do a better check than this?
+  const create = location.pathname.includes('/people/create') ? true : false;
+
+  let personData = {};
+
+  (!create) && (
+    personData = {
+      first_name: 'Name',
+      last_name: 'Last',
+      birthday: '2012-03-04',
+      gender: 'male',
+      location: 'here',
+      first_met: '2001-01-01',
+      how_we_met: 'idk',
+      interests: 'a',
+      organisation: 'job co.',
+      social_media: [],
+      image: 'iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAIAAAD/gAIDAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAI2SURBVHhe7dpBcgFBFIdxspqlJTtu4hhuwC0cgx23MMexdAPLyavqri6FZOYfyWuv8v0WySBJma9aT6cZd103wjAf+TsGIJaAWAJiCYglIJaAWAJiCYglIJaAWAJiCYglIJaAWAJiCYglIJaAWAJiCYglIJaAWAJiCf5LrLZtF4vF+IHdeTwe8w/16iI7nU7L5fJwOOTbN+yh+XyeT/JbTdPk3+kTO9Z0OrWztQGSTvtnNptN/nN9gsUaPl6eGt7lqQCxXgn0Yp077xtLavS7Ub7yprGs1GQyySUelDT5ttdl6k1jPY6pp2MnP+YV603XWev1Oh2URrvdLt1TUexP/pVFg89Z8O+OgFgCYgmIJQgcq23bfOQl8NVwNptdLpd0zNWwRylla7F08NcCjyznRZZhghdEjeU/u5uoL0P/2d1EHVn+s7uJOrL8Z3fDBC8IGWu/3+cjZzaMw0nvgJnVapXvchFyzioT1vV6bZomHTuIHcv5ycebszyXo/efkAg3sjyXo1bqfD7nGxFHludytLzJlMQbWfZqSAf+z5xFqSBYrCqbDUWwl2GVzYYi2MiqstlQBBtZFWd3wwQvIJaAWAJiCSLFqrbnV9hlJYpae35FpKVDWTc47/kVIWPVes5M8AJiCYgliBSryqR+K1Ks7XZrX6vsNyTxtpUrYs4SEEtALAGxBMQSEEtALAGxBMQSEEtALAGxBMQSEEtALAGxBMQSEEtALAGxBMQSEEtALAGxBhuNPgHlJKV46HCjogAAAABJRU5ErkJggg==',
+      encounters: [],
+      time_updated: '0002-02-02',
+    });
+
+  async function getPersonData() {
+    try {
+      personData = await apiCalls.getPerson(id);
+    } catch (err) {
+      // console.log(err);
```

Comment:
Why is line 47 commented out, after we catch the error?

Reply:
The API calls aren't properly implemented yet. Having a console log probably isn't too helpful either. Perhaps later, a proper message to the user is displayed. I guess for MVP a console log might be okay.

Reply:
For consistency with the other functions in this file, let's have the error log to the console for now.